.github/workflows/test-aks.yaml

```diff
@@ -283,14 +286,12 @@ jobs:
       - name: Deploy app
         if: steps.gen-id.outputs.RUN_TEST == 'true'
         id: deploy-app
-        run: rad deploy ${{ matrix.path }} ${{ matrix.deployArgs }} -e ${{ matrix.env }}
-      - name: Wait for all pods to be ready
```

Comment:
Do we no longer care about waiting for the pods to be ready?

Reply:
I think we discussed this in one of our 1:1 meeting. `rad deploy` should wait for all pods to be ready before returning successfully. Is that wrong?